omas/tests/failed_imports.py

```diff
@@ -40,7 +42,7 @@
     from botocore.exceptions import NoCredentialsError
     import boto3
 
-    if not os.path.exists(os.environ.get('AWS_CONFIG_FILE', os.environ['HOME'] + '/.aws/config')):
+    if not os.path.exists(os.environ.get('AWS_CONFIG_FILE', pathlib.Path.home() / '/.aws/config')):
```

Comment:
```suggestion
    if not os.path.exists(os.environ.get('AWS_CONFIG_FILE', pathlib.Path.home() + '/.aws/config')):
```

Reply:
Uhh it was not typo. It was correct. I have rewrote both parts which are using `pathlib`